packages/hooks/src/hooks/useTokenBalance/useTokenBalance.ts

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import { ethers } from 'ethers';
+import { Web3Context } from '../../Provider';
+import ERC20 from './ERC20.json';
+
+export const useTokenBalance = (address: string, owner: string) => {
```

Comment:
not sure what naming conventions are common here, but maybe `contractAddress`? I could also see us wanting `ownerAddress` instead of `owner`, unless we also accept ENS domains for the owner string

Reply:
I think the obvious the function names the better. So I also think that  ownerAddress and contractAdrees are the better options for this. 
But haven't tested the hook with an ens domain yet so I don't know how it behaves in this case.

Reply:
it probably doesn't support ENS unless you add it. I think we've implemented ENS support in other hooks/components already, so maybe it makes sense to pull that code out into a util/hook and reuse it here too. ENS support isn't blocking though

Reply:
i think util for ens would make sense here. I can have a look at how to do this best after this one is merged.